sympy/integrals/tests/test_integrals.py

```diff
@@ -1166,6 +1166,10 @@ def test_issue_8901():
     assert integrate(tanh(1.0*x)) == 1.0*x - 1.0*log(tanh(1.0*x) + 1)
     assert integrate(tanh(x)) == x - log(tanh(x) + 1)
 
+@slow
```

Comment:
How slow is this test?

Reply:
timeit measured it at 23.4 seconds on my EC2 instance (nano). I found it to be a lot of time to wait for an integral to come back unevaluated.

Reply:
I think just passing any keyword argument to Derivative is enough for the test.

Reply:
I agree; since this was the actual cause of the error, I replaced the test by one that tests that. And moved it to test_function because all this isn't about integrals; I'm not changing anything in the integrals module.